packages/now-cli/src/util/index.js

```diff
@@ -450,11 +450,11 @@ export default class Now extends EventEmitter {
         host = host.slice(0, -1);
       }
 
-      const url = `/v3/now/hosts/${encodeURIComponent(
+      const url = `/v10/now/deployments/get?url=${encodeURIComponent(
         host
-      )}?resolve=1&noState=1`;
+      )}&resolve=1&noState=1`;
 
-      const { deployment } = await this.retry(
+      const deployment = await this.retry(
```

Comment:
Can you remove the fetch on line 482 or is the version selection still necessary?

Reply:
@styfle Yes you are right, we can remove it because we already make the call to Deployments API on line 457.

Reply:
I guess we need to keep this as-is to support Now 1.0 deployments which use an older version of api-deployments.